src/java/org/apache/cassandra/service/StorageServiceMBean.java

```diff
@@ -814,6 +814,10 @@ default int upgradeSSTables(String keyspaceName, boolean excludeCurrentVersion,
     public void enforceConcurrentValidatorsLimit();
     public boolean isConcurrentValidatorsLimitEnforced();
 
+    int getConcurrentIndexBuilders();
+
```

Comment:
super nit: Get rid of the extra newline? (At least looking at the other examples...)